keyboard shortcut

audrey-grayson opened the pull request:
Added the ability to set a keyboard shortcut (by default, Command+Shift+Y) to pop tabs to the right into a new window.

audrey-grayson committed:
added keyboard shortcut

0xdevalias commented:
Thanks for the PR @OliverGray Looks good from a casual skim, but need to make some time to check things out before I merge it.

Do you know how hard it is (if possible) to make the bound key configurable from an 'options' page? This makes it easy to change for a user building their own extension, but not so good from the extension store. I think I would want to have that ability included before I merged/added this to the store build.

audrey-grayson commented:
It's actually already in there by default, in chrome://extensions you click on the top left hamburger and go to "keyboard shortcuts" and can edit all the bind keys for your extensions. By the way, I love this extension, it's so useful!

If you wanted to have a specific options page for this app it seems doable but I'm not totally sure how, I'm pretty new to Chrome extensions.

0xdevalias commented:
Ah, that is cool. Never even looked at that page before!

Docs ref: https://developer.chrome.com/extensions/commands

Eventually I would like to rewrite/expand on this extension a bit, but forever doing too many other things. So if I ever get around to it, and it makes sense to, that's when i'll look at including an options page.

**I see that you've added the manifest for `newWindowWithTabsToRight` but not `newWindowWithThisAndTabsToRight`. If you could add that as well, then i'd be happy to merge this.**

Will then need to make time/sit down at some point and figure out the release process on the app store again to get it up there. Has been a while.. :p

Thanks for the contribution (and the kind words)!

audrey-grayson committed:
added keyboard shortcut for current tab option

Ctrl+Shift+Y will now open a new window with the current tab and ones to the right, while Ctrl+Shift+U will open a new window with only tabs on the right.

audrey-grayson commented:
Added a manifest for the current tab option. I find myself using the "tabs to the right" option more than the current one... supporting that function must have slipped my mind. Oops!

0xdevalias commented:
Hehe, all good :)

Will merge now, and hopefully re-remember the upload process over the weekend! Thanks for the PR! It's awesome to hear how things I made for myself are useful to others too :3

0xdevalias commented:
@OliverGrayson Just bumped the version on the chrome store with your changes included. Thanks again!
